src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,73 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onAddMovie: (movie: Movie) => void,
+  isInList: boolean,
+  resetIsInList: () => void,
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onAddMovie,
+  isInList,
+  resetIsInList,
+}) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setIsLoading(true);
+
+    const movieFromServer = await getMovie(query);
+
+    if ('Error' in movieFromServer) {
+      setHasError(true);
+      setIsLoading(false);
+    } else {
+      const newMovie = {
+        title: movieFromServer.Title,
+        description: movieFromServer.Plot,
+        imgUrl: movieFromServer.Poster === 'N/A'
```

Comment:
GJ 🔥 
Everything is working and looks great. But the only thing that looks weird for me is this. You're creating new movie, but you already (probably) have movieFromServer. You can just add it to your list